ansible/roles/atmo-realvncserver/tasks/main.yml

```diff
@@ -123,6 +123,14 @@
   tags:
     - kill-vnc
 
+- name: Enable unencrypted connections to RealVNC
+  lineinfile:
+    dest: /etc/vnc/config.custom
+    state: present
+    backrefs: yes
+    regexp: "^SecurityTypes=RA2"
+    line: "SecurityTypes=RA2,VncAuth"
```

Comment:
When guacamole is disabled we might want to disable this task from running as well..